MailCore/API/Endpoint.swift

```diff
@@ -241,4 +241,12 @@ public extension Endpoint {
     static func attachmentToForward(uuid: String) -> Endpoint {
         return .draft(uuid: uuid).appending(path: "/attachmentsToForward")
     }
+
+    static func calendarReply(resource: String) -> Endpoint {
+        return .resource(resource).appending(path: "/reply")
```

Comment:
You can remove the return statement, and do the same in the file.

Reply:
We always use `return` in this file